<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrmSetting</class>
 <widget class="QDialog" name="FrmSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Resources.qrc">
    <normaloff>:/icons/Resources/img/settings.svg</normaloff>:/icons/Resources/img/settings.svg</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Database connection</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Template Folder Path</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Header Spaces Lines</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Letter Pad Type</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdi_databaseConnection"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_TemplateFolderPath"/>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEdit_HeaderSpaceLines"/>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="comboBox_LetterPadType"/>
     </item>
     <item row="4" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pushButton_setSettings">
         <property name="text">
          <string>Set Settings</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
